src/components/manage/Widgets/NumberWidget.jsx

```diff
@@ -46,6 +47,7 @@ const NumberWidget = (props) => {
         max={maximum || null}
         value={value || defaultValue}
         placeholder={placeholder}
+        step={step || 1}
```

Comment:
slipped change? Does this belong here?

Reply:
I added this to have support for "opacity" filter, as it can go from 0 to 1 with a step of 0.1 (and in general the step is 1), and I also made use of this widget, as we do not have a slider one.